internal/logs/aggregate/traefik_test.go

```
@@ -130,8 +134,8 @@ func TestBucketTraefik_JSONFormatPopulatesExtras(t *testing.T) {
 		`{"ClientHost":"5.6.7.8","DownstreamStatus":500,"RequestMethod":"POST","RequestHost":"api.example.com","RouterName":"router-b@docker","RequestPath":"/y","Duration":40000000,"StartUTC":"2026-05-07T11:55:00Z","TLSVersion":"1.2"}`,
 	}, "\n")
 	now := time.Date(2026, 5, 7, 12, 0, 0, 0, time.UTC)
-	d := BucketTraefik(nil, now.Add(-time.Hour), now, models.Range1h, fakeGeo{})
-	d = BucketTraefikRaw(logs, now.Add(-time.Hour), now, models.Range1h, fakeGeo{})
+	d := BucketTraefik(nil, now.Add(-time.Hour), now, models.Range1h, fakeGeo{}, testSystemStats())
```

golangci-lint (2.12.2)

[error] ineffectual assignment to d

(ineffassign)